dlc-wallet/src/main/scala/org/bitcoins/dlc/wallet/models/DLCDAO.scala

```diff
@@ -132,8 +132,8 @@ case class DLCDAO()(implicit
 
     def keyIndex: Rep[Int] = column("key_index")
 
-    def oracleSigOpt: Rep[Option[SchnorrDigitalSignature]] =
-      column("oracle_sig")
+    def oracleSigsOpt: Rep[Option[Vector[SchnorrDigitalSignature]]] =
+      column("oracle_sigs")
```

Comment:
Do we want an `executedOutcomeOpt`?

Reply:
We could but it'd normally be something we calculate anyways